src/pages/home/base/mobile-core/tabs/api-reference.md

```diff
@@ -409,6 +409,176 @@ static func getSdkIdentities(completion: @escaping (String?, Error?) -> Void)
  }];
 ```
 
+<Variant platform="android" api="initialize" repeat="14"/>
+
+#### Initialize with App ID
+
+##### Java
+
+**Syntax**
+
+```java
+ public static void initialize(
+            @NonNull final Application application,
+            @NonNull final String appId,
+            @Nullable final AdobeCallback<?> completionCallback) 
+```
+* _application_ The Android Application instance.
+* _appID_ The mobile property environment ID configured from the Data Collection UI.
+* _completionCallback_ An Optional [AdobeCallback](#adobecallback) triggered once initialization is complete.
+
+**Example**
+
+```java
+public class CoreApp extends Application {
+   @Override
+   public void onCreate() {
+      super.onCreate();
+      MobileCore.initialize(this, "ENVIRONMENT_ID", o -> {
+          Log.d(LOG_TAG, "AEP Mobile SDK is initialized");
+      });
+   }
+}
+```
+
+#### Initialize with InitOptions 
+
+##### Java
+
+**Syntax**
+
+```java
+public static void initialize(
+            @NonNull final Application application,
+            @NonNull final InitOptions initOptions,
+            @Nullable final AdobeCallback<?> completionCallback)
+```
+* _application_ The Android Application instance.
+* _initOptions_ The [InitOptions](#initoptions) to configure the SDK.
+* _completionCallback_ An Optional [AdobeCallback](#adobecallback) triggered once initialization is complete.
+
+**Example**
+
+```java
+public class CoreApp extends Application {
+   @Override
+   public void onCreate() {
+      super.onCreate();
+
+      // Use InitOptions to disable automatic lifecycle tracking.
+      InitOptions initOptions = InitOptions.configureWithAppID("ENVIRONMENT_ID")
+      initOptions.lifecycleAutomaticTrackingEnabled = false
+      MobileCore.initialize(this, initOptions, o -> {
+          Log.d(LOG_TAG, "AEP Mobile SDK is initialized");
+      });
+   }
+}
+```
+
+<Variant platform="ios" api="initialize" repeat="26"/>
```

Comment:
This tab is quite long as it contains two examples in each Swift and Objective-C. To improve readability, what do you think about splitting the languages to their own tabs as I did with the following example?
https://developer.adobe.com/client-sdks/home/base/mobile-core/lifecycle/#register-lifecycle-with-mobile-core

Reply:
It will not be consistent with other code snippets on this page, but I'll split the syntax and examples for `initialize(appId)` and `initialize(initOptions)` .